Routes/GetBirthdayPerson.js

```diff
@@ -0,0 +1,22 @@
+const { Contacts } = require("../data/Contacts")
+
+
+// req.params.birthdate
+const GetBirthdayPerson = {
+    path: '/api/birthday',
+    method: 'get',
+    execute: () => {
```

Comment:
a gente precisa receber uma data de aniversário como parâmetro nessa rota

```suggestion
    execute: (req) => {
```